src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -202,7 +203,7 @@ const ConsentGroupReview = hh(class ConsentGroupReview extends Component {
                 prev.current = current;
                 prev.future = future;
                 prev.futureCopy = futureCopy;
-                prev.isAdmin = component.isAdmin;
+                prev.isAdmin = Storage.getCurrentUser() != null ? Storage.getCurrentUser().isAdmin : false;
```

Comment:
This extra boilerplate, if carried through, would result in a ton of changes throughout the app (just search on `component.isAdmin`). This also looks unrelated to the nav bar conversion. I suggest these changes be reverted and then re-implemented throughout the app in a way that makes sense, doesn't require so much boilerplate, etc.